src/StorageSync/StorageSync/Common/StorageSyncClientWrapper.cs

```diff
@@ -41,9 +42,15 @@ namespace Microsoft.Azure.Commands.StorageSync.Common
     public class StorageSyncClientWrapper : IStorageSyncClientWrapper
     {
         /// <summary>
-        /// The kailani application identifier
+        /// The azure cloud kailani application identifier
         /// </summary>
-        public static Guid KailaniAppId = new Guid(StorageSyncResources.KailaniApplicationId);
+        private static Guid AzureCloudKailaniAppId = new Guid(StorageSyncResources.AzureCloudKailaniAppId);
```

Comment:
this is OK, but these don't need to be resource strings, since they are not user-facing (the reason we make things resource strings is so they can be translated, if necessary)